### DISCUSSION TITLE
Captains Log Values in regular builder?

### DISCUSSION THREAD

**Discussion by pawnnolonger** [bcholmes/StarTrek2d20#170] [Q&A]
Will the Captain's Log Values be added to the regular builder eventually?

**Comment by bcholmes** [comment]
I hadn't thought about it, but I don't see why not.

**Reply by pawnnolonger** [reply]
Thats the one part of character creation I hate coming up with Values.

**Reply by Felderburg** [reply]
https://www.modiphius.net/en-us/blogs/news/sta-dev-blog-004-a-guide-to-star-trek-adventures
https://nerdist.com/article/how-to-create-compelling-values-for-your-star-trek-adventures-rpg-character/
https://continuingmissionsta.com/2019/12/13/4-tips-to-create-values-that-actually-matter-in-star-trek-adventures-rpg/

I've found the above to be useful, especially where they point out that values should be both positive and negative, or can be used both ways.
